bin/export-import-data

```diff
@@ -43,7 +43,7 @@ sub export {
     if (@categories) {
         my @contacts = $body->contacts->search({
             category => { -in => \@categories },
-        })->all;
+        })->search(undef, { order_by => 'category' })->all;
```

Comment:
Is there a reason we need two `search`es here? Would this work instead:
```suggestion
            order_by => 'category',
        })->all;
```

Reply:
order_by has to be in the second parameter to search(), it's not a query parameter, but yes, not sure why I hadn't spotted the search above!

Reply:
Oops, of course.